tests/cupyx_tests/scipy_tests/fft_tests/test_realtransforms.py

```diff
@@ -0,0 +1,178 @@
+import numpy as np
+import pytest
+
+import cupyx.scipy.fft  # noqa
+from cupyx.scipy import fft as cp_fft
+from cupy import testing
+
+try:
+    # scipy.fft is available since scipy v1.4.0+
+    import scipy.fft as scipy_fft  # noqa
+except ImportError:
+    scipy_fft = None
+    pass
+
+# used to avoid SciPy bug in complex dtype cases with output_x=True
+# https://github.com/scipy/scipy/pull/11904
+scipy_cplx_bug = not cupyx.scipy.fft._scipy_150
+
+if cupyx.scipy.fft._scipy_160:
+    # additional normalization options available for SciPy>=1.6
+    all_dct_norms = [None, 'ortho', 'forward', 'backward']
+else:
+    all_dct_norms = [None, 'ortho']
+
+
+@testing.parameterize(
+    *testing.product(
+        {
+            'n': [None, 0, 5, 15],
+            'type': [1, 2, 3, 4],
+            'shape': [(9,), (10,), (10, 9)],
+            'axis': [-1, 0],
+            'norm': ['ortho'],
+            'overwrite_x': [False],
+            'function': ['dct', 'dst', 'idct', 'idst'],
+        }
+    )
+    # test all overwrite_x and norm combinations on a smaller subset of shapes
+    + testing.product(
+        {
+            'n': [None, 15],
+            'type': [2, 3],
+            'shape': [(10, 9)],
+            'axis': [-1, 0],
+            'norm': all_dct_norms,
+            'overwrite_x': [False, True],
+            'function': ['dct', 'dst', 'idct', 'idst'],
+        }
+    )
+
+)
+@testing.gpu
+@testing.with_requires('scipy>=1.4')
+class TestDctDst():
```

Comment:
```suggestion
class TestDctDst:
```

Reply:
Thanks. I had removed unittest.TestCase but should have removed the () as well.